packages/ui/.storybook/Welcome/welcome.css

```diff
@@ -8,3 +8,7 @@
   height: 100vh;
   padding: 3rem;
 }
+
+.welcome__heading {
+  font-size: xx-large;
```

Comment:
TIL about `xx-large` and the other possible values, lol.
🆒

Reply:
Me too 🧐